web/styles/subscriptions.css

```diff
@@ -647,11 +642,6 @@ h4.user_group_setting_subsection_title {
             margin-top: 2px;
             line-height: 1.5;
         }
-
-        .bottom-bar > div {
-            display: inline-block;
-            vertical-align: bottom;
-        }
```

Comment:
It looks like the history is these were added in ae7fe85ec51820fec365dae84a99d6c47dc61354, and should have been deleted in 333b8b095c7e3303bc81cdf5c761e71a29b60bbd.